src/tls/Makefile

```diff
@@ -244,15 +246,37 @@ tls-lax: $(LIBFILES) ConcreteFlags $(FSTS)
 wc:
 	wc -l $(LIBFILES) $(FSTS)
 
+# ----------------------- FFI callback library -------------------------
+FFICallbacks.cmi: ../../libs/ml/FFICallbacks.mli
```

Comment:
what about:
```
FFICallbacks.cmi FFICallbacks.cmo FFICallbacks.cmx: ../../libs/ml/FFICallbacks.mli
  $(OCAML) $(OCAMLOPTS) $<
```

and then -I ../../libs/ml everywhere so that `ocamlopt` picks up the .cmi and the .cmx?

This assumes just an .mli, no .ml